lib/entities/QuoteResponse.ts

```diff
@@ -67,31 +69,25 @@ export class QuoteResponse implements QuoteResponseData {
     );
   }
 
-  public static fromRFQ(args: FromRfqArgs): ValidatedResponse {
+  public static async fromRFQ(args: FromRfqArgs, provider?: ethers.providers.JsonRpcProvider, log?: Logger): Promise<ValidatedResponse> {
     const { request, data, type, metadata } = args;
-    let validationError: ValidationError | undefined;
+    let validationErrors: string[] = [];
```

Comment:
Why not do "ValidationError[]", I much prefer the loose typing that we had already over arbitrary strings

Reply:
The ValidtionErrors would all have the same value for the `value` field. (`data`). The only thing different is the error strings which is why I collected them as an array of strings.

Reply:
Ah I see, hm that's a bit tricky to deal with